### DISCUSSION TITLE
Fix for Zookeeper issue -3988

### DISCUSSION THREAD

**Discussion by sgv0007** [strimzi/strimzi-kafka-operator#8772] [Q&A] [answered]
Hi Team ,

We are using Strimzi 0.32 version and recently encountered below zookeeper issue which has been fixed in zookeeper version 3.6.4 and above .

Jira:  https://issues.apache.org/jira/browse/ZOOKEEPER-3988

 We have plan to upgrade strimzi version to 0.35 but zookeeper version supported for 0.35 is 3.6.3.
as per link  https://strimzi.io/docs/operators/0.35.1/deploying.html#ref-kafka-versions-str .

Can you please let us know if any plan to upgrade zookeeper or how to handle issue mentioned in jira ticket.

Thanks,
Shiva

**Comment by ppatierno** [comment, marked as answer]
The Strimzi project doesn't bring its own version of ZooKeeper.
It brings whatever is the version brought by the supported Kafka in a specific Strimzi version.
The next Strimzi 0.36.0 release (which should happen in the next couple of weeks) brings Kafka 3.4.1 and Kafka 3.5.0 which are the first versions with ZooKeeper 3.6.4. It will also have Kafka 3.4.0 (still with ZooKeeper 3.6.3).
So in order to have the new ZooKeeper 3.6.4, you should wait for Strimzi 0.36.0 to be released and then upgrading your Kafka cluster to (at least) Kafka 3.4.1 (or maybe better the latest 3.5.0).

**Reply by scholzj** [reply]
Also, just to add for the record - it seems to break way more things than it really fixes. So you might regret it once you have it ;-).

**Reply by ppatierno** [reply]
Fair enough Jakub :-D
